openqa-label-known-issues

```diff
@@ -105,10 +105,10 @@ handle_unknown() {
     local testurl=$1 out=$2 reason=$3
     to_review+=("$testurl ${reason:0:50}")
     echoerr "$testurl : Unknown issue, to be reviewed -> $testurl/file/autoinst-log.txt"
-    echoerr -e "Likely the error is within this log excerpt, last lines before shutdown:\n---"
+    echoerr -e "Likely the error is within this log excerpt, last lines before shutdown:\n  # --- 8< ---"
     # Look for different termination points with likely context
-    (grep -A 12 'Backend process died, backend errors are reported below in the following lines' "$out" || grep -B 10 'sending magic and exit' "$out" || grep -B 5 'killing command server.*because test execution ended through exception' "$out" || grep -B 5 'EXIT 1' "$out" || grep -B 10 '\(Result: died\|isotovideo failed\)' "$out" || echo '(No log excerpt found)') | head -n -1 >&2
-    echoerr "---"
+    (grep -A 12 'Backend process died, backend errors are reported below in the following lines' "$out" || grep -B 10 'sending magic and exit' "$out" || grep -B 5 'killing command server.*because test execution ended through exception' "$out" || grep -B 5 'EXIT 1' "$out" || grep -B 10 '\(Result: died\|isotovideo failed\)' "$out" || echo '(No log excerpt found)') | sed 's/^/  # /' | head -n -1 >&2
+    echoerr "  # --- >8 ---"
```

Comment:
I like the scissors. But maybe we should just use Markdown style with "```"?

Reply:
Markdown won't address the ambiguity